src/app/signOut/views/session-timeout.ejs

```diff
@@ -0,0 +1,21 @@
+<div class="govuk-width-container">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <br>
+            <h2 class="govuk-heading-xl">
+                Sorry, your session has<br>timed out
+            </h2>
+
+            <h2 class="govuk-heading">Your session automatically ends if you don't do<br>anything for 20 minutes</h2>
+            
+            <h2 class="govuk-heading">We do this to keep your information safe.<h2/>
+
+            <p class="govuk-body">To use this service again, you need to <a id="signin" href="">sign in</a></p>
```

Comment:
please add `class="govuk-link"` to the anchor element `<a class="govuk-link" id="signin" href="">sign in</a>`